src/CommunityToolkit.Maui.Core/Essentials/FolderPicker/FolderPickerImplementation.android.cs

```diff
@@ -17,7 +17,7 @@ async Task<Folder> InternalPickAsync(string initialPath, CancellationToken cance
 	{
 		if (!OperatingSystem.IsAndroidVersionAtLeast(26) && !string.IsNullOrEmpty(initialPath))
 		{
-			Trace.WriteLine("Specifying an initial path is only supported on Android 26 and later.");
+			Trace.TraceError("Specifying an initial path is only supported on Android 26 and later.");
```

Comment:
Please do not use `TraceError` in the library. It causes crash reporting tools like sentry.io to report it as an unhandled exception. Leading to our library causing erroneous bug reports for our users.